src/NServiceBus.Transport.RabbitMQ.TransportTests/When_modifying_incoming_headers.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace NServiceBus.Transport.RabbitMQ.TransportTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.TransportTests;
+    using NUnit.Framework;
+
+    public class When_modifying_incoming_headers : NServiceBusTransportTest
```

Comment:
@Particular/nservicebus-maintainers should we also add those tests to the core transport tests?

Reply:
Makes sense. That said it would mean that some transports that doesn't honor this will break when we update them to core 7.1 but I guess it's good to know and we can always disable the test for that specific transport?

Reply:
> but I guess it's good to know and we can always disable the test for that specific transport?

that was my idea as well. Once this is fixed in core, we can get rid of those tests again easily.

Reply:
:shipit: